### DISCUSSION TITLE
How to rename the tab?

### DISCUSSION THREAD

**Discussion by dprat0821** [graphql/graphiql#2916] [Q&A / Support] [answered]
I can't find any way to rename the session tabs, either by double clicking the tab or by right click action list. The tab names stay with the default `<untitled>`.
The issue is  reproducible locally or from official [live demo](http://graphql.org/swapi-graphql), accessing from desktop Chrome or Safari from MacOS 13
Any help is appreciated!

**Comment by jonathanawesome** [comment, marked as answer]
Hey there @dprat0821! Tab names are tied to the name of the first operation in the operation editor. Naming your operations is good practice, and required in some situations, so we key off of that.

This behavior is the same in most GraphQL IDEs (I can't think of one that allows a user to edit a tab name manually), and unlikely to change in GraphiQL. That said, if you have a design proposal for this interaction, I'd be happy to review!

**Reply by kluzzebass** [reply]
The now discontinued GraphQL Playground project let you change the names of tabs by double clicking them (the label then turned into an input field), so there is precedence for this functionality.
